packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -74,7 +74,9 @@ export const Alert = React.forwardRef<HTMLDivElement, Props>(
           </Box>
 
           <Box paddingY={1} grow={1}>
-            <Text weight="medium">{title}</Text>
+            <Text color="primary" weight="medium">
```

Comment:
This change probably requires some explanation;

We are displaying an `<Alert>` in the `subTitle` part of the `<Page>` component (see screenshot in the PR).

The styles that are applied to it are setting `color: <secondary>` and those are inherited by the `<Alert>` component and will set the title and body text colors to something with insufficient contrast.

Reply:
Alternatives to consider:

1. We can wrap the Alert component where we place it in the page's subtitle like so

```
<Text color="primary"><Alert ... /></Text>
```

2. We can also move the alerts for provisioned / federated / errored rules to the content of one particular tab (or all of them?) 

This has the downside of it not being visible if users share a specific tab with each other where we decided not to place the alert unless we place the alert on _all_ tabs.

3. We don't use the `<Alert />` component at all and maybe move the error to a `ToggleTip` that would be attached to the state indicator in the title – less discoverable but also feasible.

Reply:
Just to confirm: This should in theory not change what alerts look like elsewhere, because it is just making explicit the color that we already want Alerts to have?

Have you double checked that other alerts look ok after this change?

Reply:
Correct, this wouldn't change how they currently look and I've checked the storybook to find regressions and found none – I wasn't entirely sure if overwriting the color of the alert text was _by design_ though.

Reply:
I don't think it is by design as far as I'm aware 🤷 This seems safe enough if you checked some other alerts, so I'd say lets go for it and keep an eye out if someone complains about it. So from the changes to grafana/ui this looks good 👍